Source/BluetoothManager.swift

```diff
@@ -342,22 +342,29 @@ public class BluetoothManager {
     /// - Parameter peripheral: `Peripheral` which is monitored for connection.
     /// - Returns: Observable which emits next events when `peripheral` was connected.
     public func monitorConnection(for peripheral: Peripheral) -> Observable<Peripheral> {
-        return monitorPeripheral(on: delegateWrapper.rx_didConnectPeripheral, peripheral: peripheral)
+        return monitorPeripheral(
+            on: delegateWrapper.rx_didConnectPeripheral.map { ($0, nil) },
+            peripheral: peripheral
+        ).map { $0.0 }
```

Comment:
Are we creating tuple to just drop it at the end?

Reply:
Yes, it is to use `monitorPeripheral` function in both connection and disconnection functions.

Reply:
I'm not convinced to this — It's right now kind of artificial way to do this generalization. Could we find some other approach to implement this? This tuple creation reason is complicated and deserves comment at least. The best would be to find other way or just drop helper function.